force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -61,15 +58,23 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of boolean values indicating whether the string starts with 'Sales', ends with 'Force', or contains 'Awesome'.
      */
     public static List<Boolean> checkStringContent(String input) {
-        // Initialize a list to store the results
-        
-        // Check if the string starts with 'Sales'
-        
-        // Check if the string ends with 'Force'
-        
-        // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        List<Boolean> resultsList = new List <Boolean>();
+        Boolean startsWithSales = false;
+        Boolean endsWithForce = false;
+        Boolean containsAwesome = false;
+        if (input.startsWith('Sales')) {
```

Comment:
Remember that input.startsWith() returns a Boolean. You can add it directly to your results as you go.